manifests/init.pp

```diff
@@ -201,7 +201,7 @@
     $command = strip(regsubst("${script_command} -o \"${fragdir}/${concat_name}\" -d \"${fragdir}\" ${warnflag} ${forceflag} ${orderflag} ${newlineflag}", '\s+', ' ', 'G'))
 
     # make sure ruby is in the path for PE
-    if defined('$is_pe') and $::is_pe {
+    if defined('$is_pe') and str2bool("${::is_pe}") {
```

Comment:
This needs `# lint:ignore:only_variable_string` at the end of the line so lint doesn't fail